src/utils/proton.js

```diff
@@ -37,34 +57,39 @@ class ProtonSDK {
     } catch (e) {
       return e;
     }
-  }
+  };
 
   logout = async () => {
     await this.link.removeSession(this.appName, this.session.auth);
     localStorage.removeItem('savedUserAuth-foobar');
-  }
+  };
 
   restoreSession = async () => {
-    const savedUserAuth = JSON.parse(localStorage.getItem('savedUserAuth-foobar'));
+    const savedUserAuth = JSON.parse(
+      localStorage.getItem('savedUserAuth-foobar')
+    );
     if (savedUserAuth) {
       try {
-        this.link = await ConnectWallet({
-          linkOptions: { chainId: this.chainId, endpoints: this.endpoints },
-          transportOptions: { requestAccount: this.requestAccount, backButton: true },
-          selectorOptions: { appName: this.appName, appLogo: Foobar, showSelector: false }
-        });
-        const result = await this.link.restoreSession(this.requestAccount, savedUserAuth);
-        if (result) {
-          this.session = result;
-          return { auth: this.session.auth, accountData: this.session.accountData[0] };
+        await this.connect(true, false);
```

Comment:
I know I said the last comment was the last one, sorry 😆 

its good practice to use named parameters through object destructuring for functions. If you're reading `this.connect(true, false)`, you don't know what the true and false are for immediately, and it becomes hard to extend them in the future.

Reply:
Ooh makes sense, thank you! Sorry just saw this! Working on taking out `showSelector` so we'll probably update these demos again! So, in this case it's best practice to destructure the invocation this `this.connect({ restoreSession: true })`? And then destructure the input as well `connect = ({ restoreSession }) => {` or just name it as a generic `options` argument `connect = (options) => {`?